tests/pkg_tinycrypt/test_utils.h

```diff
@@ -0,0 +1,125 @@
+/*  test_utils.h - TinyCrypt interface to common functions for tests */
```

Comment:
Even if imported as is, this file should be adapted to RIOT's code standards. Otherwise the static tests fail.

Reply:
If it is used as is, maybe adding the package's folder to the include path saves us from dealing with this?